src/routes/products.js

```diff
@@ -0,0 +1,69 @@
+const express = require("express");
+let router = express.Router();
+let products = require("../db/all-products.json");
+
+router.get("/*", (req, res) => {
+  if (req.url === "/") {
+    res.setHeader("Content-Type", "application/json");
+    res.send(products);
+    res.end();
+    return;
+  }
+
+  if (req.url.includes("?")) {
```

Comment:
better to check actually for `req.query.ids`. Or even better would be to move ids logic to a middleware